utils.py

```diff
@@ -0,0 +1,18 @@
+import os
+import yaml
+from categories.models.category import Category
+
+
+def get_base_category():
+    """
+    Function to get the category instance with slug
+    equal to the service name
+    """
+    config_path = os.path.join('services', 'helpcentre', 'config.yml')
+    with open(config_path, 'r') as f:
+        app_configuration = yaml.safe_load(f)
+    service_name = app_configuration['nomenclature']['name']
+    service_verbose = app_configuration['nomenclature']['verboseName']
+    cat_obj, created = Category.objects.get_or_create(
+        slug=service_name, name=service_verbose)
```

Comment:
Line break between arguments, closing parenthesis must be on its own line.

Reply:
Added :+1: